pipeline/terraform0.12/terraform.tf

```diff
@@ -0,0 +1,90 @@
+provider "aws" {
+  region = "eu-west-1"
```

Comment:
Might be worth pinning the version here?

Reply:
👍